api/v1beta1/frrconfiguration_types.go

```diff
@@ -79,6 +79,16 @@ type Router struct {
 	// Prefixes is the list of prefixes we want to advertise from this router instance.
 	// +optional
 	Prefixes []string `json:"prefixes,omitempty"`
+
+	// Imports is the list of imports we want for this router / vrf
```

Comment:
`Imports is the list of imports`, could be more readable to like is a list of VRFs from which to import their prefixes?